docs/docs/zh/guide/datasource/oracle.md

```diff
@@ -0,0 +1,12 @@
+![image](https://user-images.githubusercontent.com/48941586/193031878-d7c35140-fe12-4d45-a0b9-a70dc018d25f.png)
```

Comment:
@Chris-Arith Could u plz put the image under `../../../../img/new_ui/dev/datasource/`? See: https://raw.githubusercontent.com/apache/dolphinscheduler/dev/docs/docs/zh/guide/datasource/hive.md as an example. BTW, please add the title for this doc. Thanks : )

Reply:
pls check